metadata_service/app.py

```diff
@@ -69,6 +69,26 @@ def handle_url_invalid(exc):
         400,
     )
 
+@app.errorhandler(HTTPException)
```

Comment:
You're right - the different codes follow the contract we have with our west partners. Werkzeug's HTTPException contains different 4xx errors so we need to check what the contract specifies and then check the corresponding [werkzeug exception classes](https://werkzeug.palletsprojects.com/en/3.0.x/exceptions/#error-classes). For other than 4xx we should return:
````
    type: "SYSTEM_ERROR",
    code: 202
````